documentation/docs/20-core-concepts/20-load.md

```diff
@@ -548,6 +548,8 @@ A `load` function that calls `await parent()` will also rerun if a parent `load`
 
 Dependency tracking does not apply _after_ the `load` function has returned — for example, accessing `params.x` inside a nested [promise](#streaming-with-promises) will not cause the function to rerun when `params.x` changes. (Don't worry, you'll get a warning in development if you accidentally do this.) Instead, access the parameter in the main body of your `load` function.
 
+Also accessing one single query parameter is tracked independently from the rest of the url. This means that accessing `event.url.searchParams.get("query")` inside a `load` function will make that load function rerun only when the `query` search param changes. For example navigating from `/search?query=svelte&page=1` to `/search?query=svelte&page=2` will not rerun a load function that access `event.url.searchParams.get("query")` but not `event.url.searchParams.get("page")`.
```

Comment:
I think this has more context when inserted after the statement on line 601.

> ### When do load functions rerun?
> To summarize, a `load` function will rerun in the following situations:
> - It references a property of `url` (such as `url.pathname` or `url.search`) whose value has changed. Properties in `request.url` are _not_ tracked
> - _It references a URL query parameter, etc.._

Reply:
I feel like it would be fitting in both places...I will definitely correct that line too, thanks

Reply:
I've added the line after 601 but for now i've not removed the previews one

Reply:
Awesome. Thanks for taking on this PR. It’s looking real good :)

Reply:
I'm glad to be able to help and i also need this sooooo bad so it's a win-win 😄